views/give.handlebars

```diff
@@ -70,8 +70,8 @@
       </div>
     </div>
 
-    <div class="col-12 {{#unless hideMap}}col-lg-4{{/unless}} order-lg-1 locations-container" style="display: none;">
-      <div id="locations-list {{#if hideMap}}w-100{{/if}}" class="locations-list"></div>
+    <div class="col-12 {{#unless hideMap}}col-lg-4{{/unless}} order-lg-1 locations-container" style="height: 100px; display: none;">
+      <div id="locations-list" class="locations-list {{#if hideMap}}w-100{{/if}}"></div>
```

Comment:
@ngottlieb I wanted to call attention to this change, since I don't know much handlebars. Looks to me like a Bootstrap css class was sometimes added as a "second id"? When not added, that left a trailing space in the id that was tripping me up.

Reply:
Hah. Yeah, 100% a typo/accident. thanks for catching it!